src/integration_sdk.js

```diff
@@ -36,8 +36,14 @@ const defaultSdkConfig = {
   commandLimiter: null,
 };
 
+/* global window, navigator, process */
+const isBrowser = typeof window !== 'undefined' && typeof window.document !== 'undefined';
+
 // User-Agent string for the SDK
-const sdkUserAgentString = `sharetribe-flex-integration-sdk-js/${sdkVersion}`;
+// For browsers, append to the browser's user agent string,
+const sdkUserAgentString = isBrowser
+  ? `${navigator.userAgent} sharetribe-flex-integration-sdk-js/${sdkVersion}`
+  : `sharetribe-flex-integration-sdk-js/${sdkVersion} (node/${process.versions.node})`;
```

Comment:
Change request: I think it's a bit too bold assumption that `navigator` exists if the `isBrowser` condition is true. And same for thing for `process`...

There are just so many JS runtimes out there (React Native, Bun, Rhino) etc that I'm not sure all of them have these globals.

Thus, I'd be on the safe side and add something like `typeof navigator !== 'undefined' ? navigator.userAgent : ""` and same for `process`.